weave/integrations/openai/openai_test.py

```diff
@@ -0,0 +1,705 @@
+import os
```

Comment:
Top tier test suite .... chefs kiss

Please take a look at `weave/tests/test_monitoring_openai.py` ... these are our current tests for openai. It would be good to ensure that we cover everything tested in those

Reply:
Checked! I think my tests are superset of `weave/tests/test_monitoring_openai.py`.